src/tests/ftest/cart/threaded_server.c

```diff
@@ -123,7 +123,8 @@ int main(int argc, char **argv)
 		return -1;
 	}
 
-	crt_context_create(&crt_ctx);
+	rc = crt_context_create(&crt_ctx);
```

Comment:
maybe better to return error rather than assert.

Reply:
It is necessary to call crt_context_create() successfully. Otherwise, we should quit in error. I think assert is proper here.